docs/discussions/301.html

```diff
@@ -0,0 +1,167 @@
+Scroll to the bottom and press <code>Import ONC</code>, then upload the onc file<br>
+Delete the file and then go to your quick access settings (click the clock) <br>
+Click <code>VPN</code>, and then the name you selected<br>
+Depending on how used to account is, or how ratedlimited it is. It should a couple seconds. If it dosen't connect right away, try again or wait until later.<br>
+<br>
+<h2>Video on Creating and Importing</h2>
+<video width="500" height="500" controls>
+    <source src="/discussions/google_screen_recording_2024-05-08T04-12_14.958Z.webm" type="video/webm">
+  Your browser does not support the video tag.
+  </video><br><br>
+<hr class="divider">
+
+<h1>Changing your file's region</h1>
+        <br>
+<h2>Getting the hostname</h2>
+        <br>
+<h3>Server Directory</h3>
+You need another hostname or port to change your file's region, these are listed <a href="/tools/directory/serverlister.html">here</a>
+<br>
+        <h3>Manual</h3>
+Or you can get this hostname from a .ovpn file.
+on the <a href="https://account.proton.me/u/6/vpn/OpenVpnIKEv2">IKEv2</a> page. Download a file from a region you want to emulate.<br>
+For example;<br>
+<img src="/discussions/328875738-9d3fa4f4-b6fb-434f-95cd-a85d4c2a1ed6.png" width=600 alt="image"/>
+<br>
+Once downloading it,
+Drag it from the ChromeOS files app into the browser, it should now open in browser
+Somewhere in the file you should see numbers in this format ##.##.###.###
+        <br>
+<h2>Changing in file creator</h2>
+Those numbers are your new hostname to put in the file creator.
+<h3>Video outlining these changes.</h3>
+<video width="500" height="500" controls>
+    <source src="/discussions/google_screen_recording_2024-05-08T12-05_14.026Z.webm" type="video/webm">
+  Your browser does not support the video tag.
+  </video><br><br>
+        <hr class="divider">
+
+<h1>Prox tools</h1>
+        <br>
+<h2>ACPROX/GoTool</h2>
+        <br>
+<h3>About</h3>
+The all new <a href="https://go.sillylittle.tech">Go Tool</a> allows you to proxy yourself into different pages using the Ultraviolet framework<br>
+<img src="https://github.com/dswan36/SillyLittleFiles/assets/75678535/e62e7bcf-d329-4971-a9c3-4c135a1cb5fe" width="600" />
+<br>
+<sub>Hey! Look, Google But... notice this distinct lack of google.com, in the address bar!</sub>
+<br><br>
+<h2>Other Tools</h2>
+        <br>
+<h3>Go Tool Details</h3>
+        <br>
+Find or make an App to use, search for quick access, or look in the settings for some fun options<br>
+<img src="https://github.com/dswan36/SillyLittleFiles/assets/75678535/c6e589da-d633-40d6-9cbb-0190dcdbf076" width="600" />
+<br><h3>Web Inspect</h3><br>
+Need to launch inspect element but don't have a supported device? Try this...
+<ol>
+  <li>Copy the code below:<br><code>(function () { var script = document.createElement('script'); script.src="https://go.sillylittle.tech/eruda.js"; document.body.append(script); script.onload = function () { eruda.init(); } })();</code></li>
```

Comment:
Consider using emgithub with a copy button here (maybe).

Reply:
LGTM for now